content/strategy-goals/strategy/code-graph/search/index.md

```diff
@@ -12,17 +12,19 @@ Quicklinks:
 
 ## Competitive landscape
 
-Our competitive landscape in the search space is sparse at the moment, but there are serious potential competitors we need to monitor–GitHub is the elephant in the room. It’s very likely they only need to release “good enough” code search to deal Sourcegraph’s search product a serious blow. While we believe our solution will remain far superior to whatever GitHub releases, there is a real risk that the switching costs of using Sourcegraph will be too high for users to switch, if GitHub’s search improves significantly.
+The biggest threat to our dominance as the leader in open-source, free, and commercially available code search is the advnet of GitHub Code search, currently widely available in technology preview. Last quarter we stated:
 
-There are also existing open source projects that may be “good enough” for many users. While we don’t believe these pose a serious threat to our enterprise business, they are competitors with our Cloud offering and, to the extent users prefer those experiences, we need to consider how to compete against them for attention and usage.
+> It’s very likely they only need to release “good enough” code search to deal Sourcegraph’s search product a serious blow. While we believe our solution will remain far superior to whatever GitHub releases, there is a real risk that the switching costs of using Sourcegraph will be too high for users to switch, if GitHub’s search improves significantly.
 
-As the scope of the Search teams expands, we'll face competition in new areas. Some examples are onboarding new developers and enabling responses to codebase changes. There are both new entrants (Swimm) and incumbents (Zapier, IFTTT) in those spaces, but we believe our unique approach to search positions us to deliver differentiated solutions by leveraging the distinct advantages of our code graph, such as its [unified experience](../index.md#deliver-a-unified-experience): [Notebooks](https://sourcegraph.com/search/notebook) as onboarding documentation can be created from code searches, browsed alongside the user’s code, and searched alongside the user’s code. [Code monitors](https://sourcegraph.com/code-monitoring) can eventually integrate with Batch changes and Code insights, to allow users to directly change the pattern they’re monitoring from Sourcegraph, and visualize instances of this pattern in their codebase over time.
+While we still maintain a strong reputation with our customers and the market, GitHub's code search is in fact "good enough" for many users and will continue to be. We retain our competitive advantage in the SMB and Enterprise markets, where we serve users with thousands of repos or more, on multiple code hosts, and even non-git VCSs. Our Cloud offering continues to see significant usage as well.
```

Comment:
Is it actually good enough for serious use cases? I'd contend it's not, and list those reasons here.